pitch.md

```diff
@@ -23,38 +31,79 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+The user might find it inconvenient that they can't see the final product of 
+their coloring page on the terminal since this program is not interactive that way.
+
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy for the user to generate coloring pages with pictures they 
+already own. They willalso be able to specify their own arbitrary settings such as removing all colorsfrom the original pictures except purple. However, it would be impossible or very difficult for the user to take a peak on the final product before the program ends. In other words, they wouldn't be able to interact with the final 
+product in real-time.
+
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I had no idea this existed before I just did some research, there's a website 
+that allows users to convert photos into coloring pages, 
+https://www.reallycolor.com/. I want my DSL to essentially do the same thing, 
+but also provide users with different settings such as removing all colorings 
+except a certain color.
+Also, I find it very inconvenient that this website doesn't offer the service 
+for free.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+Developing language aspects of the project would be time-consuming since 
+manipulating colors and other things of photos can be hard. But I don't imagine 
+this project needing to integrate with any APIs. And the system part would be 
+less time-consuming, since most of it would be converting the picture into a 
+sketched veresion of it, possibly in a pdf.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this project is ambitious for me in a way that it's going to require a 
+lot of image manipulation, which I've always found hard.
+
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This project is suitable as a DSL class project, since it has a very narrow and 
+specific domain. However, the scope of the project might be a barrier because it
+ will be very time-consuming and require me to do a lot of self-learning 
+ beforehand about image manipulation.
```

Comment:
Agreed, although using APIs to help with image manipulation could allow you to devote more time to the DSL aspects and additional customizations/features.